apps/dashboard/src/components/workflow-list.tsx

```diff
@@ -158,7 +160,18 @@ export const WorkflowList = () => {
                             <FaCode className="size-3" />
                           </Badge>
                         )}
-                        <TruncatedText text={workflow.name} />
+                        <TruncatedText
+                          className="cursor-pointer"
+                          text={workflow.name}
+                          onClick={() => {
+                            navigate(
+                              buildRoute(ROUTES.EDIT_WORKFLOW, {
+                                environmentId: currentEnvironment?._id ?? '',
+                                workflowId: workflow._id,
+                              })
+                            );
+                          }}
+                        />
```

Comment:
I wasn't sure if this is what it should be because, from the designs, it's not clear. But decided to do it like when clicking on the workflow name, redirect the user to the workflow editor.